KmsClient.js

```diff
@@ -57,11 +64,13 @@ export class KmsClient {
     _assert(type, 'type', 'string');
     _assert(invocationSigner, 'invocationSigner', 'object');
 
-    const keySuite = cryptoLd.suites.get(type);
-    const contextUrl = keySuite.CONTEXT_URL;
+    let SUITE_CONTEXT = symmetric.get(type);
+    if(!SUITE_CONTEXT) {
+      ({SUITE_CONTEXT} = cryptoLd.suites.get(type));
+    }
```

Comment:
`cryptoLd.suites` is just a map defined here: https://github.com/digitalbazaar/crypto-ld/blob/master/lib/CryptoLD.js#L14

This flow runs into two issues, if we do hit the codepath on L69, and the type is not in the `suites` map, I think we get a `cannot destructure `SUITE_CONTEXT` from undefined` sort of error.

Then, depending on how that is resolved, we might still have a condition where `SUITE_CONTEXT === undefined` leading to `undefined` being included in the operation's `@context` below.

One way or another, we need to throw a meaningful error if the `SUITE_CONTEXT` cannot be found.

Reply:
We still have an issue on L72 where an error is going to be thrown if `cryptoLd.suites.get` returns `undefined`.  The destructured assignment will fail with an error.

The `else` conditional that has been added here is going to throw a `NotFoundError` if `SUITE_CONTEXT` is truthy on L71 which means that it was found successfully on L70.

I suggest something along these lines:
```
    let suiteContextUrl = symmetric.get(type);
    if(!suiteContextUrl) {
      ({SUITE_CONTEXT: suiteContextUrl} = cryptoLd.suites.get(type) || {});
      if(!suiteContextUrl) {
        throw new Error(`Unknown key type: "${type}".');
      }
    }
```

Reply:
addressed here fd0033e